.python-version

```diff
@@ -0,0 +1 @@
+renuo-thumbor-2.7.9
```

Comment:
Yes it is. This is where pyenv works different than rbenv: in Ruby, you have one version of Ruby installed on your system, and gems are installed and managed with bundler and the Gemfile. When an app is started in Ruby, the gems (including the versions!) are extracted from the Gemfile.lock.

In Python it works differently: you install the Python version on the system, but after this you create a own environment per project (called virtualenv or venv or virtualenvwrapper etc.). Next you install the dependencies needed (often from the requirements.txt file). After that, you run python, and it will run with the libraries installed in the virtualenv. There is no Gemfile.lock to freeze the gems, or to load the correct version when Python starts (just like before bundler in Ruby was created). Therefore, every project needs it's own virtualenv.

Therefore, in Python, you need to activate a certain virtualenv per project. Using this name in the .python-version will execute the following command (more or less): ```pyenv virtualenv activate renuo-thumbor-2.7.9```. In a Ruby project, this command would be: ```rbenv shell 2.2.2``` (because in Ruby, the Gemfile.lock will handle the versioning of the gems when the ruby program is started).

Example:

```
# this creates the virtualenv
pyenv virtualenv 2.7.9 renuo-thumbor-2.7.9
# this installs all libs / dependencies
pip install -r requirements.txt
# now when you run "python", the installed versions are loaded. If you have two different projects 
# with different dependency versions, and would use the same virtualenv, there would be a problem.
```

See also: http://fgimian.github.io/blog/2014/04/20/better-python-version-and-environment-management-with-pyenv/